README.md

```diff
@@ -12,6 +12,20 @@
 - ruby (v2.3.1)
 - bundler (v1.12.5)
 
+## setup
+
+```bash
```

Comment:
The `README.md` file should contain only information on how to run the application locally.

Would you create another README, e.g. README-docker-development.md, for this need?